idd/Energy+.idd.in

```diff
@@ -4502,6 +4503,20 @@ Schedule:Constant,
        \type real
        \default 0
 
+Schedule:File:Shading,
+       \min-fields 1
+       \memo A Schedule:File:Shading points to a text computer file that has
+       \memo hourly/sub-hourly sunlit fraction data for all exterior surfaces.
+  A1 , \field File Name
+       \memo The name of the file that writes all shading data.
+       \required-field
+       \retaincase
+  A2 ; \field File Format
+       \type choice
+       \key CSV
+       \key JSON
```

Comment:
Wait, what?  Can this do JSON??  If this is just a placeholder option for later that isn't implemented, then it needs to go away.

Reply:
Yes. It's a placeholder as we discussed for the NFP. I can remove it.

Reply:
During the NFP stage, I wanted the code to be written so that later it would be easy to add JSON.  We can't have things in the IDD that the user sees that aren't true.  Sorry for the confusion.